cdap-ui/server/express.js

```diff
@@ -151,6 +151,41 @@ function makeApp (authAddress, cdapConfig, uiSettings) {
 
   });
 
+  // This is used for single click deploy app from Cask Market
+  app.get('/streamApp', function(req, res) {
```

Comment:
Yes, it requires the link of source and link to destination.. Just need to rename this route to something more appropriate. Any suggestion?